src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -67,7 +67,33 @@ def _post(self, request: Request):
                 full_config_requested=full_config_requested,
             )
         else:
-            return Response("Unsupported version, we only support versions 1 to 3.", 400)
+            return Response("Unsupported version, we only support versions 1 to 4.", 400)
+
+    def _get_v4_config(self, request):
+        res = self._post_or_schedule_by_key(request)
+        metrics.incr("relay.project_configs.post_v4.pending", amount=len(res["pending"]))
+        metrics.incr("relay.project_configs.post_v4.fetched", amount=len(res["configs"]))
+
+        if request.GET.get("global"):
+            res["global"] = {
+                "measurements": res["measurements"],
+                "metricsConditionalTagging": res["metricsConditionalTagging"],
+            }
```

Comment:
What do you think about having a separate function for the global config, and merge it? I think something like the following could work:
```suggestion
        if request.GET.get("global"):
            res["global"] = _get_global_config(...)

# ...

def _get_global_config():
  return {
    "measurements": get_measurements_config(...),
    "metricsConditionalTagging": get_metrics_conditional_tagging_rules(...)
  }
```

Another benefit of isolating the global config computation is that it's easier to test it. What do you think?